Apps/WebClient/src/ClientApp/src/utility/errorTranslator.ts

```diff
@@ -79,7 +76,6 @@ export default class ErrorTranslator {
     }
 
     private static getServiceName(serviceName: string): string {
-        console.log(serviceName);
```

Comment:
at one point, we may need to come up with some best practices for logging on client side i.e. where these logs should be written and why :-)